apps/vr-tests/package.json

```diff
@@ -15,8 +15,7 @@
     "start": "just-scripts dev:storybook"
   },
   "devDependencies": {
-    "@fluentui/eslint-plugin": "*",
-    "@fluentui/make-styles-webpack-loader": "9.0.0-beta.4"
```

Comment:
We don't need this dependency anymore. `@griffel/webpack` is defined in root `package.json`.